lfucg-exactions/server/plats/utils.py

```diff
@@ -0,0 +1,71 @@
+from .models import Lot, Plat
+from accounts.models import *
+
+def calculate_lot_balance(lot_id):
+    lot = Lot.objects.filter(id=lot_id)[0]
+
+    payments = Payment.objects.filter(lot_id=lot_id)
+
+    sewer_payment = 0
+    non_sewer_payment = 0
+
+    total_exactions = (lot.dues_roads_own +
```

Comment:
It would be more efficient to calculate total_exactions as sewer_exactions + non_sewer_exactions after calculating those two